frontend-ssr/components/search/Search.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Nitpick: the proper name should be capitalized - `components/Search` - just like `Header` and `Footer`.